unittests/snapshot_tests.cpp

```diff
@@ -388,10 +388,10 @@ void compatible_versions_test()
    std::filesystem::copy(source_log_dir / "blocks.index", config.blocks_dir / "blocks.index");
```

Comment:
The `abieos` submodule update should be via a PR to `abieos`. I don't see one.

Reply:
As the changes were small and I'd to have a quick test, I went ahead making the abieos changes in the `sync_call` branch. I will make sure all the future changes will be done from a branch created out of the `sync_call` branch.